docs/rules/singleline-html-element-content-newline.md

```diff
@@ -0,0 +1,102 @@
+# require a line break before and after the contents of a singleline element (vue/singleline-html-element-content-newline)
+
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule enforces a line break before and after the contents of a singleline element.
+
+
+:-1: Examples of **incorrect** code:
+
+```html
+<div attr>content</div>
+
+<tr attr><td>{{ data1 }}</td><td>{{ data2 }}</td></tr>
+
+<div attr><!-- comment --></div>
+```
+
+:+1: Examples of **correct** code:
+
+```html
+<div attr>
+  content
+</div>
+
+<tr attr>
+  <td>
+    {{ data1 }}
+  </td>
+  <td>
+    {{ data2 }}
+  </td>
+</tr>
+
+<div attr>
+  <!-- comment -->
+</div>
+```
+
+## :wrench: Options
+
+```json
+{
+  "vue/singleline-html-element-content-newline": ["error", {
+    "strict": false,
+    "ignoreNames": ["pre", "textarea"]
+  }]
+}
+```
+
+- `strict` ... if false, if there are no attributes on the element, this rule allows having contents in one line.  
```

Comment:
What do you think about renaming `strict` option that I proposed earlier to something a bit more descriptive, like `ignoreWhenNoAttribures`?
```
- `ignoreWhenNoAttributes` ... allows having contents in one line, when given element has no attributes. default `true`

Reply:
I think that the name `ignoreWhenNoAttribures` is easy to understand and very good!